CHANGELOG/CHANGELOG-2.2.md

```diff
@@ -0,0 +1,159 @@
+# Release notes for v2.2.0
+
+[Documentation](https://kubernetes-csi.github.io)
+# Changelog since v2.1.0
+
+## Urgent Upgrade Notes
+
+### (No, really, you MUST read this before you upgrade)
+
+- For drivers that support CSI Migration, a "migrated" label was added to the csi_sidecar_operations_seconds metric that indicates if the call is from the migration path. Metric collectors should be updated with the new field. ([#560](https://github.com/kubernetes-csi/external-provisioner/pull/560), [@nearora-msft](https://github.com/nearora-msft))
+
+## Changes by Kind
```

Comment:
I think we'll also want a deprecations section for https://github.com/kubernetes-csi/external-provisioner/issues/602

Reply:
Done